lib/crypto/key.go

```diff
@@ -18,17 +18,21 @@ func ReadPub64(suite abstract.Suite, r io.Reader) (abstract.Point, error) {
 // WritePub64 writes a public point to a base64 representation
 func WritePub64(suite abstract.Suite, w io.Writer, point abstract.Point) error {
 	enc := base64.NewEncoder(base64.StdEncoding, w)
-	err := suite.Write(enc, point)
-	enc.Close()
-	return err
+	return write64(suite, enc, point)
+}
+
+func write64(suite abstract.Suite, wc io.WriteCloser, data ...interface{}) error {
+	errW := suite.Write(wc, data)
+	if err := wc.Close(); err != nil && errW == nil {
+		return err
+	}
+	return errW
```

Comment:
Yeah but your `WriteCloser` comes from the base64 itself, and not from a file or whatever (what we give to `WritePub64` is only a `Writer`, and you create the `WriterCloser` on top of it with base64). So there's no resource non-freed except the internal buffer of the base64 encoder, so that will be GC'd anyway.